model/utils.go

```diff
@@ -193,6 +193,64 @@ func IsValidTeamName(s string) bool {
 	return true
 }
 
+var validTeamNameCharacter = regexp.MustCompile(`^[a-z0-9-]$`)
+
+func CleanTeamName(s string) string {
```

Comment:
The IsValidTeamName function was in here so I just added it next to it. I didn't want to have to move this code and the unit test code. I can move it if you feel it's important